zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -0,0 +1,139 @@
+            if logical_pool_name in lp.logicalPoolName:
+                physical_pool_name = lp.physicalPoolName
+                break
+
+    if physical_pool_name is None:
+        raise Exception('cannot found logical pool[%s], you must create it manually' % logical_pool_name)
+
+    return physical_pool_name
+
+
+def create_volume(logical_pool_name, lun_name, lun_size):
+    return shell.call("%s create file --path %s/%s --size %s --stripecount %d --stripeunit %s --user %s --format json" % (DEFAULT_ZBS_BIN_PATH, logical_pool_name, lun_name, lun_size, DEFAULT_STRIPE_VOLUME_COUNT, DEFAULT_STRIPE_VOLUME_UINT, DEFAULT_ZBS_USER_NAME))
```

Ruff (0.8.2)

`shell` may be undefined, or defined from star imports

(F405)